Src/BootCamp.Chapter/ContactsCenter.cs

```diff
@@ -1,16 +1,42 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.IO;
 using System.Text;
 
 namespace BootCamp.Chapter
 {
     public class ContactsCenter
     {
-        private readonly List<Person> _people;
+        public List<Person> People { get; } = new List<Person>();
 
         public ContactsCenter(string peopleFile)
         {
-            // load people
+            if (!File.Exists(peopleFile)) throw new FileNotFoundException();
+            var fileContent = File.ReadAllLines(peopleFile);
+            if (fileContent.Length < 1) throw new ArgumentOutOfRangeException();
+
+            CreatePeopleDatabase(fileContent[1..]);
+        }
+
+        private void CreatePeopleDatabase(string[] fileContent)
+        {
+            foreach (var person in fileContent)
+            {
+                var personInfo= person.Split(",");
+
+                if (person.Contains('\"')) personInfo = SplitQuotaCase(personInfo);
+                if (personInfo.Length != 7) throw new ArgumentOutOfRangeException(nameof(personInfo));
+                
+                var name = personInfo[0];
```

Comment:
Could have privavate enum nested to avoid nasty indexing and reader guessing what the column actually is.